apps/sync/test/unit/TestAdapterMock.js

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
Test files are not licensed with MPL2. Use Public Domain instead, please.

https://www.mozilla.org/en-US/MPL/headers/